src/sentry/features/temporary.py

```diff
@@ -275,6 +275,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("projects:similarity-embeddings", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
     manager.add("projects:similarity-embeddings-grouping", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
     manager.add("projects:similarity-embeddings-metadata", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("projects:similarity-embeddings-remove-seer-grouping-record-table-init", ProjectFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
adding a feature flag to allow s4s to drop the existing (incorrect) records table and not re-initialize it so that we can try backfilling it with the correct information [here](https://github.com/getsentry/seer/blob/506aae671c35396b8c928073a3ad3a1d55d6ae79/src/seer/grouping/grouping.py#L98). we will eventually want to remove this janky initialization logic for all orgs, but we want to start with s4s

Reply:
talked offline, it is best to do database updates like this through migrations or a script instead of in realtime in an endpoint if possible